core/editor/loader/v2/editor-v2-loader.php

```diff
@@ -188,4 +191,23 @@ private function get_styles() : array {
 			->unique()
 			->all();
 	}
+
+	/**
+	 * Get additional dependencies for packages that rely on global variables, rather
+	 * than an explicit npm dependency (e.g. `window.elementor`, `window.wp`, etc.).
+	 *
+	 * @param string $handle
+	 *
+	 * @return string[]
+	 */
+	private function get_additional_deps( string $handle ): array {
+		$map = [
+			'elementor-v2-editor-v1-adapters' => [
+				'elementor-editor',
+				'elementor-web-cli',
```

Comment:
```suggestion
				'elementor-web-cli',
```

If the editor will be loaded it will do side effect

Reply:
+ we need to add filter

Reply:
Removed